src/App.js

```diff
@@ -2,13 +2,17 @@ import React, { Suspense } from "react";
 import { BrowserRouter as Router } from "react-router-dom";
 import { defaultLightTheme, useTheme, Spinner } from "pi-ui";
 import Routes from "./Routes";
+import ErrorsBoundary from "src/components/ErrorBoundary";
 
 const App = () => {
   useTheme(defaultLightTheme);
   return (
     <Router>
       <Suspense fallback={<Spinner invert />}>
-        <Routes />
+        <ErrorsBoundary>
+          <Routes />
+        </ErrorsBoundary>
+        >
```

Comment:
Hey mate. Right now, the `Results.jsx` component is where we throw the error from the server. If we put the `ErrorBoundary` on the `Page.jsx`, as it should be, it doesn't catch the error because the `Results.jsx` component doesn't render anything, not even `Page.jsx`. This will be fixed in the upcoming PR regarding the results page, and `ErrorBoundary` will be placed correctly. This is just temporary, so that it still catches errors once this PR goes in.